torat_server/api.go

```diff
@@ -0,0 +1,119 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
```

Comment:
I use https://echo.labstack.com/ most of the time, are you set on mux or would it be ok for you to switch?